string id
string robot_link
string object_link